### ISSUE TITLE
Make the bird for our Flappy Bird Game

### ISSUE THREAD

**Issue by priyanshisharma** [GirlScript-Ireland/Gaming-Booster#8]
This is recommended to be solved after #7 

So we need a bird to be shown, and we're using a circle for that. Lets make our bird visible. Use the following steps to do so.
1. Navigate to `bird.pde` file int the `flappy_bird` folder of `Game Coders`.
2. Make a `class Bird` and steps are to be followed inside the class.
3. Declare a `PVector pos;` to hold the position of the Bird and `float r = 16;` for the radius of our circle.
4. Make the constructor for the `Bird` and initialise `pos` as follows
```
Bird() {
      pos = new PVector(50, height/2); 
    }
```
5. Declare a `void show()` function and use the following statements to make your bird.
(i) `ellipse(pos.x,pos.y,r*2, r*2);` for a circle.
(ii) `fill(224, 218, 34);` to add colour to the circle. (Change the colour according to your wish)
(iii) `stroke(0,0,0);` to add border to the circle.
(iv)`strokeWeight(2);` thickness of the circle's border. 

**OPTIONAL (We will do it later if not now)**
Try displaying your bird by actually calling the show function in `flappy_bird`'s `draw()` function. Hint: You'll need to create an object `Bird b;`, initialise it in the `setup()` as `b = new Bird();` and call `b.show();` in the `draw()` function.
